util/constants.ts

```diff
@@ -2,4 +2,5 @@ export enum MANDATARIS_STATUS {
   BEEINDIGD = 'http://data.vlaanderen.be/id/concept/MandatarisStatusCode/b8866fa2-d61c-4e3d-afaf-8a29eaaa7fb9',
   DRAFT = 'http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/588ce330-4abb-4448-9776-a17d9305df07',
   EFFECTIEF = 'http://data.vlaanderen.be/id/concept/MandatarisStatusCode/21063a5b-912c-4241-841c-cc7fb3c73e75',
+  BEKRACHTIGD = 'http://data.lblod.info/id/concept/MandatarisPublicationStatusCode/9d8fd14d-95d0-4f5e-b3a5-a56a126227b6',
```

Comment:
Since this is not an actual mandataris status but a mandataris publication status, I would separate these.

Reply:
draft, effectief and bekrachtigd are publication statuses, beeindigd is an actual mandataris status

Reply:
Split up the status into mandataris status and publication status